client/a8c-for-agencies/sections/marketplace/shopping-cart/shopping-cart-menu/item.tsx

```diff
@@ -16,6 +19,8 @@ export default function ShoppingCartMenuItem( { item, onRemoveItem }: ItemProps
 	const translate = useTranslate();
 	const userProducts = useSelector( getProductsList );
 
+	const siteId = getQueryArg( window.location.href, 'site_id' )?.toString();
```

Comment:
Sorry for the follow-up comment. It might be better if we don't check for the `siteId` URL query parameter here to determine when to show the Remove button. We can maybe allow `onRemoveItem` to be null or undefined and make it as a way to determine when to hide the button. wdyt?

Reply:
Makes total sense. Thanks!

I have updated it here: https://github.com/Automattic/wp-calypso/pull/89675/commits/3e3388140f6512ecd6ffbec776f03ccbeccdb78d